atintegrators/DeltaQPass.c

```diff
@@ -161,11 +176,12 @@ void mexFunction(	int nlhs, mxArray *plhs[], int nrhs, const mxArray *prhs[])
         R2=atGetOptionalDoubleArray(ElemData,"R2"); check_error();
         T1=atGetOptionalDoubleArray(ElemData,"T1"); check_error();
         T2=atGetOptionalDoubleArray(ElemData,"T2"); check_error();
+        chrom_maxorder=atGetOptionalLong(ElemData,"chrom_maxorder", 5); check_error();
```

Comment:
Same comment on the default value also here in the mexFunction. It should be 1, not 5.

Reply:
done